phase2/ignition/vanilla/kubelet.service

```diff
@@ -1,6 +1,8 @@
 [Unit]
 Description=Kubernetes Kubelet Server
 Documentation=https://github.com/kubernetes/kubernetes
+After=docker.service
+Requires=docker.service
```

Comment:
Good catch!  Docker in```Photon OS``` have dependency on ```network-online.target``` and it may not be true with other OS. Yet, I didn't understand that why this is replacement of ```docker.service``` and not adding dependency on both. Can you please explain this?